content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md

```diff
@@ -0,0 +1,191 @@
+v1.29.0 will be released in December. The CRI-O community will also **not**
+support release branches older than `release-1.28`, because there have been CI
+requirements merged into `main` which could be only backported to `release-1.28`
+with appropriate efforts.
+
+For example, If an end user would like to install the latest available version
+of the CRI-O `main` branch, then they can add the repository in the same way as
+they do for Kubernetes.
+
+### `rpm` Based Distributions
+
+For `rpm` based distributions, users can run the following commands as `root`
+to install CRI-O:
+
+#### Add the Kubernetes repo
+
+```bash
+cat <<EOF | tee /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://pkgs.k8s.io/core:/stable:/v1.28/rpm/
+enabled=1
+gpgcheck=1
+gpgkey=https://pkgs.k8s.io/core:/stable:/v1.28/rpm/repodata/repomd.xml.key
+EOF
+```
+
+#### Add the CRI-O repo
+
+```bash
+cat <<EOF | tee /etc/yum.repos.d/cri-o.repo
+[cri-o]
+name=CRI-O
+baseurl=https://pkgs.k8s.io/addons:/cri-o:/prerelease:/main/rpm/
+enabled=1
+gpgcheck=1
+gpgkey=https://pkgs.k8s.io/addons:/cri-o:/prerelease:/main/rpm/repodata/repomd.xml.key
+EOF
+```
+
+#### Install official package dependencies
+
+```bash
+dnf install -y \
+    conntrack \
+    container-selinux \
+    ebtables \
+    ethtool \
+    iproute \
+    iptables \
+    socat
+```
+
+#### Install the packages from the added repos
+
+```bash
+dnf install -y --repo cri-o --repo kubernetes \
+    cri-o \
+    kubeadm \
+    kubectl \
+    kubelet
+```
+
+#### Enable and start CRI-O
+
+```bash
+systemctl enable --now crio.service
+```
+
+### `deb` Based Distributions
+
+For `deb` based distributions, they can run as `root` user:
+
+#### Install dependencies for adding the repositories
+
+```bash
+apt-get update
+apt-get install -y software-properties-common curl
+```
+
+#### Add the Kubernetes repository
+
+```bash
+curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.28/deb/Release.key |
+    gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
+echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.28/deb/ /" |
+    tee /etc/apt/sources.list.d/kubernetes.list
+```
+
+#### Add the CRI-O repository
+
+```bash
+curl -fsSL https://pkgs.k8s.io/addons:/cri-o:/prerelease:/main/deb/Release.key |
+    gpg --dearmor -o /etc/apt/keyrings/cri-o-apt-keyring.gpg
+echo "deb [signed-by=/etc/apt/keyrings/cri-o-apt-keyring.gpg] https://pkgs.k8s.io/addons:/cri-o:/prerelease:/main/deb/ /" |
+    tee /etc/apt/sources.list.d/cri-o.list
+```
+
+#### Install the packages
+
+```bash
+apt-get update
+apt-get install -y cri-o kubelet kubeadm kubectl
+```
+
+#### Enable and start CRI-O
+
+```bash
+systemctl enable --now crio.service
+```
+
+The `prerelease:/main` project prefix in the CRI-O package paths can be replaced
+with `stable:/v1.28`, `stable:/v1.29`, `prerelease:/v1.28` or `prerelease:/v1.29`
+if another package stream should be used.
```

Comment:
```suggestion
The Project's `prerelease:/main` prefix at the CRI-O's package path, can be replaced with 
`stable:/v1.28`, `stable:/v1.29`, `prerelease:/v1.28` or `prerelease:/v1.29`
if another package stream should be used.
```
Line 162: Clarify the meaning, did you wanted to say 
"If another package, stream should be used" or "If another stream package is used."